shopfloor_mobile/static/wms/src/services/odoo.js

```diff
@@ -67,16 +67,19 @@ export class OdooMixin {
         console.log(response.status, response.statusText, response.url);
         return this._error_info(response, json);
     }
-    _get_headers() {
+    _get_headers(method) {
         // /!\ IMPORTANT /!\ Always use headers w/out underscores.
         // https://www.nginx.com/resources/wiki/start/
         // topics/tutorials/config_pitfalls/#missing-disappearing-http-headers
-        return {
-            "Content-Type": "application/json",
+        let headers = {
             "SERVICE-CTX-MENU-ID": this.process_menu_id,
             "SERVICE-CTX-PROFILE-ID": this.profile_id,
             "API-KEY": this.apikey,
         };
+        if (method == "POST") {
+            headers["Content-Type"] = "application/json";
```

Comment:
Why this condition? Are we returning something else than json on `GET`?

Reply:
this is what we send and on GET you don't send any JSON.